bin/GoogleAdsCleanupServices.php

```diff
@@ -48,11 +48,18 @@ class GoogleAdsCleanupServices {
 	 */
 	protected $code_path = null;
 
+	/**
+	 * @var string[] List of Service to NOT remove even when usage is not found.
+	 */
+	protected $avoid_cleanup = [
```

Comment:
Some services are loaded directly via `use` functions. However, sometimes we call them using `ResourceNames` helper. For that cases we add here the Service to avoid its removal.